src/main/java/org/kohsuke/github/GHCommit.java

```diff
@@ -42,11 +42,19 @@ public GitUser getAuthor() {
             return author;
         }
 
+        public Date getAuthoredDate() {
+            return GitHub.parseDate(author.date);
+        }
+
         @WithBridgeMethods(value = GHAuthor.class, castRequired = true)
         public GitUser getCommitter() {
             return committer;
         }
 
+        public Date getCommitDate() {
+            return GitHub.parseDate(author.date);
```

Comment:
So what is the difference of this two methods? Copy-paste instead `commit.date`?